client/src/utils/GetMeetingLink.test.ts

```diff
@@ -135,4 +136,19 @@ describe('getMeetingLink', () => {
     const result = makeRoomsJoinUrl('mockRoomId', 'mockUserId');
     expect(result).toBe('/visit?roomId=mockRoomId&userId=mockUserId');
   });
+
+  test('should return true if RoomsMeetingUrl is valid', () => {
```

Comment:
lets add another test case to cover the "false" case as well

Reply:
i.e. if the link:
- does not start with window.location.origin + '/visit'
- has no roomId query parameter
- has no userId query parameter
- query parameters are invalid format: `http://localhost:3000/visit?123=roomId&userId=456` and `http://localhost:3000/visit?roomId=123&456=userId`

Reply:
Test coverage currently being 100% I think we should skip this - 
![image](https://user-images.githubusercontent.com/100614160/208759365-e9c89ef0-3e2a-46b5-97a1-1225f94c2c8b.png)

Reply:
I am surprised that it is showing 100% when it is clear that we are not covering that path. I think it is because the success path still passes through the entire `if` block.

After organizing the tests by putting each unit in its own `describe` block, we can see that this file is actually not 100% covered:
![2022-12-20 13_29_40-GetMeetingLink test ts - communication-services-virtual-visits-js - Visual Studi](https://user-images.githubusercontent.com/43504546/208771417-f284fe09-c483-4b7c-af66-d75d426783be.png)

Please feel free to refer to/copy-paste the changes here:
- https://github.com/Azure-Samples/communication-services-virtual-visits-js/commit/d4381a56f985e885d0e28e6283ff5bf740a4f857

This adds the m
![2022-12-20 13_48_03-GetMeetingLink test ts - communication-services-virtual-visits-js - Visual Studi](https://user-images.githubusercontent.com/43504546/208773833-b5f83a31-7d11-438c-b080-b6f2af02712a.png)
issing test cases for this file, and organizes each function in its own describe block.

Reply:
Oh nice! great find.. Can you push your changes to this same branch.. We can merge it out quicker..

Reply:
Sure thing. I hope it would still allow me to approve this though. We shall see...